lib/search.js

```diff
@@ -3,6 +3,17 @@ const agent = require('./helpers/https-agent');
 const handleResponse = require('./helpers/handle-response');
 const extractSource = require('./helpers/extract-source');
 
+
+const EXCLUDE_BLOG_POSTS = {
+	bool: {
+		must_not: {
+			term: {
+				type: 'live-blog-post'
+			}
+		}
+	}
+};
```

Comment:
Having zero knowledge of Elasticsearch indexes and what kind of things have an impact on query speed: would adding this to almost every query run against Elasticsearch have a significant performance impact across the Customers Products estate in aggregate?

Reply:
I am going to show how limited my know of Elasticsearch and its terminology is right now..

Certain properties are "indexed" (not sure if this is ES terminology) so querying based on these properties doesn't have such a large impact on performance. `type` is one of those properties within our schema so this shouldn't have a huge impact.

Elasticsearch provides a `took` property in the response which tells us how long the query took to process on its side. I ran a basic search query to return items tagged with a concept and then the same again but filtering out anything with the `type` that equals `live-blog-posts`. This added 1 millisecond to the `took` time going from 2 to 3.

Example query 
```
{
		"query": {			
			"bool": { 
				"must_not": {
					"term": {
						"type": "live-blog-post"
					}
				},
				"filter": [ 
					{ "term": { "annotations.id": "dfb721f5-5003-4b5e-af38-9b55c574b50e" }}
				]
			}
    }
}
```